package.json

```diff
@@ -14,6 +14,7 @@
     "postinstall": "npm --prefix server install && npm --prefix client install",
     "auditDep": "npm audit fix && npm --prefix server audit fix && npm --client server audit fix",
     "prepare": "husky install",
+    "test": "npm --prefix server test && CI=true npm --prefix client test -- --passWithNoTests",
```

Comment:
Can I check if the -- before --passWithNoTests is required? What does it do?

Reply:
`--` passes the arguments after it to the command invoked by the npm run script, so it's needed